### PR TITLE
Remove dead code from funind.

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

**Comment by maximedenes** [comment]
Many `glob_constr` refactorings hit this code, which is actually unused.

File: plugins/funind/invfun.ml [hunk 1]
```diff
@@ -194,10 +184,9 @@ let rec generate_fresh_id x avoid i =
     id::(generate_fresh_id x (id::avoid) (pred i))
 
 
-(* [prove_fun_correct functional_induction funs_constr graphs_constr schemes lemmas_types_infos i ]
+(* [prove_fun_correct _induction funs_constr graphs_constr schemes lemmas_types_infos i ]
```
Comment (SkySkimmer) [inline plugins/funind/invfun.ml]: ??

**Review by SkySkimmer — APPROVED** [review]

File: plugins/funind/invfun.ml (hunk 1 above)
Comment (maximedenes) [reply plugins/funind/invfun.ml]: Well spotted :)

**Comment by SkySkimmer** [comment]
How did you find that this was dead?

**Comment by maximedenes** [comment]
> How did you find that this was dead?

It didn't make enough sense to be used :)

**Comment by maximedenes** [comment]
So by hand, not using an automated tool, if that was the question.

**Pull request merged**